chekjun/cadet-frontend
# Cg

### Description
Added the control buttons.

### Type of change

### How to test
Deploy the front-end. Check that the GitHub control buttons now have options for "Open", "Save", "Save As" and "Log In".
